tomviz/DataSource.cxx

```diff
@@ -497,6 +497,40 @@ void DataSource::setSpacing(const double spacing[3])
   emit dataPropertiesChanged();
 }
 
+void DataSource::setActiveScalars(const char* arrayName)
```

Comment:
I feel like the API would fit better if it used QString, and then created the const char* for the VTK function.

Reply:
Agreed. Changed in latest revision.